subsys/net/lib/lwm2m/Kconfig

```diff
@@ -56,6 +56,16 @@ config LWM2M_ENGINE_MESSAGE_HEADER_SIZE
 	help
 	  Extra room allocated to handle CoAP header data
 
+config LWM2M_COAP_MAX_MSG_SIZE
+	int "LWM2M CoAP maximum message size"
+	default LWM2M_COAP_BLOCK_SIZE
+	range 64 8168
```

Comment:
8kB is 8192

Reply:
The header takes at least 24 bytes. But of course this could be left for the build time assert to check.

Reply:
Boundaries removed.